units/commit_queue.go

```diff
@@ -114,6 +115,17 @@ func (j *commitQueueJob) Run(ctx context.Context) {
 	}
 	j.AddError(errors.Wrap(cq.SetProcessing(true), "can't set processing to true"))
 
+	// log time waiting in queue
+	timeWaiting := time.Now().Sub(nextItem.EnqueueTime)
+	grip.Info(message.Fields{
+		"source":       "commit queue",
+		"job_id":       j.ID(),
+		"item":         nextItem,
+		"time_waiting": timeWaiting,
```

Comment:
This will log the # of nanoseconds since 1970 - you may want to log the string() method so that it's readable at a glance in splunk

Reply:
We often use `.Seconds()`